test_utils/factories.py

```diff
@@ -350,39 +350,41 @@ class Meta:
     is_revoked = False
 
 
-class EnterpriseCustomerCatalogFactory(factory.django.DjangoModelFactory):
+class EnterpriseCatalogQueryFactory(factory.django.DjangoModelFactory):
     """
-    EnterpriseCustomerCatalog factory.
+    EnterpriseCatalogQuery factory.
 
-    Creates an instance of EnterpriseCustomerCatalog with minimal boilerplate.
+    Creates an instance of EnterpriseCatalogQuery with minimal boilerplate.
     """
 
     class Meta:
         """
-        Meta for EnterpriseCustomerCatalog.
+        Meta for EnterpriseCatalogQuery.
         """
 
-        model = EnterpriseCustomerCatalog
+        model = EnterpriseCatalogQuery
 
+    title = factory.Faker('sentence', nb_words=4)
     uuid = factory.LazyAttribute(lambda x: UUID(FAKER.uuid4()))
-    enterprise_customer = factory.SubFactory(EnterpriseCustomerFactory)
 
 
-class EnterpriseCatalogQueryFactory(factory.django.DjangoModelFactory):
+class EnterpriseCustomerCatalogFactory(factory.django.DjangoModelFactory):
     """
-    EnterpriseCatalogQuery factory.
+    EnterpriseCustomerCatalog factory.
 
-    Creates an instance of EnterpriseCatalogQuery with minimal boilerplate.
+    Creates an instance of EnterpriseCustomerCatalog with minimal boilerplate.
     """
 
     class Meta:
         """
-        Meta for EnterpriseCatalogQuery.
+        Meta for EnterpriseCustomerCatalog.
         """
 
-        model = EnterpriseCatalogQuery
+        model = EnterpriseCustomerCatalog
 
-    title = factory.Faker('sentence', nb_words=4)
+    uuid = factory.LazyAttribute(lambda x: UUID(FAKER.uuid4()))
+    enterprise_customer = factory.SubFactory(EnterpriseCustomerFactory)
+    enterprise_catalog_query = factory.SubFactory(EnterpriseCatalogQueryFactory)
```

Comment:
ah this line is why the order switch happened. It took me a bit to catch that the diff was just reordering those factories.

Reply:
I was gonna make a comment on the PR... but yeah... had to reference the query in the catalog factory so the switch was needed 😞